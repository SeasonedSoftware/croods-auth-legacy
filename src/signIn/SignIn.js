import React from 'react'
import New from '../New'

export default props => {
  const { relativePath = '/auth/sign_in' } = props

  return <New {...props} name="auth.signIn" path={relativePath} />
}
