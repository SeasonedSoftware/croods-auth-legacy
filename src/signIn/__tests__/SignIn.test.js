import React from 'react'
import renderer from 'react-test-renderer'

import SignIn from '../SignIn'

jest.mock('../../New', () => props => <div {...props}>New</div>)

it('renders correctly', () => {
  const props = { foo: 'bar' }
  const tree = renderer.create(<SignIn {...props} />).toJSON()

  expect(tree).toMatchSnapshot()
})
