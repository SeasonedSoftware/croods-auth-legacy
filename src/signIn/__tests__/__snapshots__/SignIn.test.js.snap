// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  foo="bar"
  name="auth.signIn"
  path="/auth/sign_in"
>
  New
</div>
`;
